app/models/reporting/filter_base.rb

```diff
@@ -137,6 +142,10 @@ def match_route_options_for_select
       end.to_h
     end
 
+    def program_options_for_select
+      Program.order(name: :asc).pluck(:name, :id).to_h
```

Comment:
calling `.to_h` here will throw out duplicates based on name.  Since the match is by name at the end of the day, (above) maybe that's ok, but it's a bit odd.  Let's leave it for now.

Reply:
I cloned this one from agency_options_for_select below, but happy to switch it up if that makes more sense.